recipes/seqan/2.x.x/conanfile.py

```diff
@@ -1,56 +1,81 @@
 import os
-from conans import ConanFile, tools
-from conans.errors import ConanInvalidConfiguration
+
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import copy, get
+from conan.tools.layout import basic_layout
+from conan.tools.scm import Version
+
+required_conan_version = ">=1.52.0"
+
 
 class SeqanConan(ConanFile):
     name = "seqan"
+    description = (
+        "SeqAn is an open source C++ library of efficient algorithms and data structures for the analysis of sequences "
+        "with the focus on biological data. Our library applies a unique generic design that guarantees high performance, "
+        "generality, extensibility, and integration with other libraries. "
+        "SeqAn is easy to use and simplifies the development of new software tools with a minimal loss of performance."
+    )
+    license = "BSD-3-Clause"
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/seqan/seqan"
-    description = """
-SeqAn is an open source C++ library of efficient algorithms and data structures for the analysis of sequences 
-with the focus on biological data. Our library applies a unique generic design that guarantees high performance, 
-generality, extensibility, and integration with other libraries. 
-SeqAn is easy to use and simplifies the development of new software tools with a minimal loss of performance.
-"""
-    topics = ("conan", "cpp98", "cpp11", "cpp14", "cpp17",
-              "algorithms", "data structures", "biological sequences")
-    license = "BSD-3-Clause"
-    settings = "compiler"
+    topics = ("algorithms", "data structures", "biological sequences", "header-only")
+
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
     no_copy_source = True
 
     @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    def _min_cppstd(self):
+        return 14
 
     @property
     def _compilers_minimum_version(self):
         return {
             "gcc": "5",
-            "Visual Studio": "14",
             "clang": "3.4",
-            "apple-clang": "3.4"
+            "apple-clang": "3.4",
+            "msvc": "190",
+            "Visual Studio": "14",
         }
 
     def configure(self):
```

Comment:
```suggestion
    def validate(self):
```
Validation should be part of `validate()` 😉